qa/rpc-tests/may152018-forkactivation_1.py

```diff
@@ -34,7 +34,8 @@ def setup_network(self, split=False):
     def run_test(self):
 
         # Advance the time beyond the timeout value
-        cur_time = int(time.time())
+        #cur_time = int(time.time())
+        cur_time = 1526400000 - 10
```

Comment:
what's wrong with using `time.time()`?

Reply:
I was having trouble with -mocktime in the new set of tests.  I suppose I could have launched here with the tweak of forktime but then i would have had to rework upper set of tests ... (as simple as they look it actual took me quite a while to get the new tests to work and didn't want to go back over and edit them - i'll look at this again later when i get back).